app/controllers/api/v1/occupation_standards_controller.rb

```diff
@@ -30,6 +30,7 @@ def create
         organization_id: current_user.employer_id,
         new_title: create_params[:title],
       )
+      @os = OccupationStandard.includes(organization: :logo_attachment).find_by(id: @os.id)
```

Comment:
@littleforest so bullet was complaining about N+1 issues with `logo_attachment` now being in the org serializer. I want to get some logos going tonight so i j ust got this working but am wondering if there's a better way? It seems silly to reload the @os with the `includes` but it didn't seem to make sense in `clone_as_unregistered!` either.

At first I thought the `options[:include]` from the fast_json api was handling eager loading but that just seems to define relationships that're included?

```
  1) API::V1::OccupationStandardsController POST #create with valid params returns new occupation standard
     Failure/Error: post path, params: params, headers: header
     
     Bullet::Notification::UnoptimizedQueryError:
       user: root
       POST /api/v1/occupation_standards
       USE eager loading detected
         Organization => [:logo_attachment]
         Add to your finder: :includes => [:logo_attachment]
       Call stack
         /usr/src/app/app/serializers/api/v1/organization_serializer.rb:10:in `block in <class:OrganizationSerializer>'
         /usr/src/app/app/controllers/api/v1/occupation_standards_controller.rb:113:in `render_resource'
         /usr/src/app/app/controllers/api/v1/occupation_standards_controller.rb:32:in `create'
         /usr/src/app/spec/requests/api/v1/occupation_standards_spec.rb:262:in `block (4 levels) in <top (required)>'
```

Reply:
...and ignore this! Broke a spec after a master merge adn ripping it out there's now no bullet warning.